### ISSUE TITLE
Missing formal Parameter

### ISSUE THREAD

**Issue by andresan21** [Eonasdan/bootstrap-datetimepicker#1882]
Hello,

I was trying to set up the datepickers linked so that the DateTo cannot be cannot be less than DateFrom. I keep getting "Missing formal parameter from my DateReplicate(). 

function DateReplicate(){
      var minDate = new Date(moment().subtract(1, 'month').calendar());
        $('.datetimepickerFrom').datetimepicker({
        	defaultDate: moment().format("L hh:mm A"),
        	showTodayButton: true,
        	useCurrent: false,
        	showClear: true,
        	sideBySide: true,
        	minDate: minDate,
        	toolbarPlacement: 'bottom',
        	widgetPositioning: {
        		horizontal: 'left',
        		vertical: 'bottom'
        	}
        });
        $('.datetimepickerTo').datetimepicker({            
        useCurrent: false, 
        showTodayButton: true,
        showClear: true,
		sideBySide: true,
        toolbarPlacement: 'bottom',
        widgetPositioning: {
        		horizontal: 'left',
        		vertical: 'bottom'
        	}
     });

	$('.datetimepickerFrom').datetimepicker({
			$('.datetimepickerTo').on("dp.change", function(e){
				$('.datetimepickerFrom').data("DateTimePicker").minDate(e.date);
			});
			$('.datetimepickerFrom').on("dp.change", function(e){
				$('.datetimepickerTo').data("DateTimePicker").maxDate(e.date);
			}); 
}

can someone tell me what I am doing wrong? Thanks! :dancer:

**Comment by Eonasdan** [comment]
Please provide a jsfiddle of your issue. You can use [this fiddle](http://jsfiddle.net/Eonasdan/0Ltv25o8/) as a base.

**Comment by andresan21** [comment]
Thanks for the quick reply! At this time I got through that error and I am facing a new one. Essentially I want to apply the link dates on all <tr>'s that are being appended to the table. At this time the only <tr> that acts as its suppose to is the first one and all other ones will not act as the first one. I assigned them as a class instead of ID's thinking I could just call the test function and make it work that way but I am obviously wrong somewhere. here is the fiddler to check this out. http://jsfiddle.net/Riftsan/o0a7540t/ Thanks again.
